crosslink/cmd/root.go

```diff
@@ -118,6 +119,23 @@ func newCommandConfig() *commandConfig {
 	}
 	c.rootCommand.AddCommand(&c.workCommand)
 
+	c.tidyCommand = cobra.Command{
+		Use:   "tidy output_path",
```

Comment:
Thinking about the name a bit more, I think this should not be called `tidy`, since, to the typical Go user, it implies the tool will do the actual tidying. Maybe `generate-tidy-list`? Or any other name that makes the purpose of this subcommand a bit more clear

Reply:
How about `crosslink tidylist`?